tpi_rs/src/middleware/persistency/binary_persistency.rs

```diff
@@ -0,0 +1,294 @@
+use std::collections::HashMap;
+use std::io;
+use std::io::Read;
+use std::io::Seek;
+use std::io::SeekFrom;
+use std::io::Write;
+use std::ops::Deref;
+use tokio::sync::watch;
+use tokio::sync::watch::Receiver;
+use tokio::sync::watch::Sender;
+use tokio::sync::RwLock;
+use tokio::time::Instant;
+
+use super::default_hash;
+use super::error::PersistencyError;
+
+const BINARY_VERSION: u32 = 1;
+const BINARY_MAGIC: u32 = 0xdeadbeef;
```

Comment:
let's come up with something more original.. or unique

```suggestion
const BINARY_MAGIC: u32 = 0xdef0ad9e;
```

Reply:
what about `b"TMAPPDB"`. This storage format is not necessarily tightly coupled to Turing Pi use-cases.